projects/plugins/jetpack/modules/tiled-gallery/tiled-gallery/tiled-gallery-layout.php

```diff
@@ -1,24 +1,80 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+
+/**
+ * Tiled gallery layout class.
+ */
 abstract class Jetpack_Tiled_Gallery_Layout {
-	// Template allow list.
+	/**
+	 * Template allow list.
+	 *
+	 * @var array
+	 */
 	private static $templates = array( 'carousel-container', 'circle-layout', 'rectangular-layout', 'square-layout' );
-	private static $partials  = array( 'carousel-image-args', 'item' );
 
-	protected $type; // Defined in child classes
+	/**
+	 * Partial list.
+	 *
+	 * @var array
+	 */
+	private static $partials = array( 'carousel-image-args', 'item' );
+
+	/**
+	 * Type of gallery - defined in parent class.
+	 *
+	 * @var string
+	 */
+	protected $type;
+
+	/**
+	 * The attachments.
+	 *
+	 * @var object
+	 */
 	public $attachments;
+
+	/**
+	 * The attachment link.
+	 *
+	 * @var string
+	 */
 	public $link;
+
+	/**
+	 * If the image is in grayscale.
+	 *
+	 * @var bool
+	 */
 	public $grayscale;
+
+	/**
+	 * How many columns.
+	 *
+	 * @var int
+	 */
 	public $columns;
-	public function __construct( $attachments, $link, $grayscale, $columns ) {
 
+	/**
+	 * Constructor function.
+	 *
+	 * @param object $attachments - the attachmed image.
+	 * @param string $link - the attachment link.
+	 * @param bool   $grayscale - if the image is in grayscale.
+	 * @param int    $columns - how many columns.
+	 */
+	public function __construct( $attachments, $link, $grayscale, $columns ) {
 		$this->attachments           = $attachments;
 		$this->link                  = $link;
-		$this->needs_attachment_link = ! ( isset( $link ) && $link == 'file' );
+		$this->needs_attachment_link = ! ( isset( $link ) && $link === 'file' );
```

Comment:
The `isset()` is pointless here, we know the variable exists because it's a parameter. So this is equivalent to `! ( $link !== null && $link === 'file' )` which we can simplify a bit further.
```suggestion
		$this->needs_attachment_link = $link !== 'file';
```